client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -0,0 +1,185 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { translate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { Card, Button } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'components/action-panel';
+import ActionPanelTitle from 'components/action-panel/title';
+import ActionPanelBody from 'components/action-panel/body';
+import ActionPanelCta from 'components/action-panel/cta';
+import CardHeading from 'components/card-heading';
+import { getTaskList } from 'lib/checklist';
+import Gridicon from 'components/gridicon';
+import getSiteChecklist from 'state/selectors/get-site-checklist';
+import isUnlaunchedSite from 'state/selectors/is-unlaunched-site';
+import { getSiteOption } from 'state/sites/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import NavItem from './nav-item';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const getTaskCopy = ( task, taskOptions ) => {
```

Comment:
@mmtr Note that we modified this to a function, since translate would only run once otherwise. (imagine switching languages without a page refresh).

Not sure if we'd like to also map actions here, feel free to rename the function if it works out.